media/js/firefox/whatsnew/experiment-whatsnew-68-fr.js

```diff
@@ -19,7 +19,7 @@
 
     // courtesy of https://davidwalsh.name/query-string-javascript
     function getUrlParam(name) {
-        name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
```

Comment:
Not every slash here is a `no-useless-escape` error.

If you look in your text editor you should see something similar to:
![Screen Shot 2019-10-07 at 8 11 59 AM](https://user-images.githubusercontent.com/30483988/66324398-2e5ef380-e8da-11e9-8431-e8145c340b63.png)
Note how one of the slashes has a double yellow underline?
![Screen Shot 2019-10-07 at 8 15 05 AM](https://user-images.githubusercontent.com/30483988/66324608-9d3c4c80-e8da-11e9-9853-5de0bfa3a292.png)
That is the only one that needs to be removed. Removing others breaks the regex and is creating test failures.

Reply:
This feedback also applies to:
- `experiment-whatsnew-68-enUS.js`
- `experiment-whatsnew-68-enGB.js`
- `experiment-whatsnew-68-enCA.js`
- `experiment-whatsnew-68-de.js`

Reply:
Ok Cool I will fix this right away..